docs/navigate.md

```diff
@@ -57,6 +60,21 @@ update itself while the user is panning around on it (set to null to disable
 intermediate pans, the plot will then not update until the mouse button is
 released).
 
+**mode** a string specifies which pan mode to be use. Accepted values:
+empty string for normal mode without pan hint or direction snapping;
+'smart' for smart mode with pan hint. The graph movement will snap to one direction
+when the drag direction is close to it;
+'smart lock' for smart mode with pan hint. The graph movement will always snap to a direction
```

Comment:
We should never have options for a property that are essentially enumerated values be strings with spaces. We should instead use standard camel-casing (i.e. 'smartLock').

Reply:
Will be fixed in the following commits.